models/model.js

```diff
@@ -9,7 +10,17 @@ const readTopics = () => {
 };
 
 
+const readArticleById = (article_id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+      .then(({ rows }) => {
+        if (rows.length === 0) {
+          return Promise.reject({ status: 404, msg: 'Article not found' });
+        }
+        return rows[0];
```

Comment:
Very good model function! Good check for empty rows to send a _Promise.reject_ and good use of the `$` binding 🏆